src/main/java/com/esri/samples/portal/oauth/README.md

```diff
@@ -1,42 +1,41 @@
-<h1>OAuth</h1>
+<h1>Authenticate with OAuth</h1>
 
-<p>Authenticate with OAuth 2.0 to retrieve the user's profile information.</p> 
+<p>Authenticate with ArcGIS Online (or your own portal) using OAuth2 to access secured resources (such as private web maps or layers). Accessing secured items requires logging in to the portal that hosts them (an ArcGIS Online account, for example).</p>
 
-<p><img src="OAuth.png"/></p>
+<p><img src="authenticate-with-oauth.png"/></p>
```

Comment:
picky but: the rest of the Java sample image names are in this format, we should be consistent

AuthenticateWithOAuth.png

Reply:
Not picky at all! Consistency is key.
This also reminds me that the screenshot needed updating. 
I'm noticing the sample class is named `OAuthSample` but everything seems to refer to `AuthenticateWithOAuth`. Should the class name be changed to match?

Reply:
If you've changed the sample title and references etc then yes, it would make sense for you to also update the class name.